scripts/utilities/vcs.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @fileoverview
+ * Provides an interface to get Version Control System (VCS) data.
+ */
+
+import gitChangedFiles from "git-changed-files";
+
+let __changedFiles = null; // Cache for git-changed-files result
```

Comment:
This cache can probably be omitted (to remove side effects). Scripts can generally be expected to request changes only once.